ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -316,6 +315,13 @@ void onAdLeftApplication(String placementId) {
       }
     }
 
+    @Override
+    void onAdClosed(String placementId) {

```

Comment:
do we need this, for case of Banner from VungleBannerAdapter can this be invoked from some place

Reply:
we need this. it's called from VungleBannerAdapter via onAdEnd. in the end we get exactly 1 invoke of onAdClosed, which is intented behavior